packages/trpc/server/document-router/schema.ts

```diff
@@ -74,12 +73,7 @@ export const ZSendDocumentMutationSchema = z.object({
     message: z.string(),
     timezone: z.string(),
     dateFormat: z.string(),
-    redirectUrl: z
```

Comment:
zod url's have issues apparently, it's probably easier to just update the refine to allow an empty string.

Reply:
Any references?

Reply:
On the OG PR

Reply:
Got it 👍